spec/unit/manifest_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'etc'
+
+module ThreeScale
+  module Backend
+    describe Manifest do
+      describe '.thread_safe?' do
+
+        it 'returns false' do
+          expect(described_class.thread_safe?).to eq(false)
+        end
+
+      end
+
+      describe '.compute_workers' do
+
+        before :each do
+          ENV::delete('PUMA_WORKERS')
+        end
+
+        it 'returns 8(value of PUMA_WORKERS_CPUMULT) * <number of CPUs> if no PUMA_WORKERS environment variable is set' do
+          expect(described_class.compute_workers(Etc.nprocessors)).to eq(Etc.nprocessors*8)
```

Comment:
Here we also want to avoid coupling to the actual value of `PUMA_WORKERS_CPUMULT`, so we should avoid any hardcoding of values. Use `described_class.const_get :PUMA_WORKERS_CPUMULT` instead of `8` and change the spec description to also avoid the actual value.

Reply:
we should not use ENV inside functions or methods. Unittesting them becomes difficult.